src/org/jgroups/util/SslContextFactory.java

```diff
@@ -43,24 +53,24 @@ public class SslContextFactory {
             log.debug("Using OpenSSL");
          }
       } catch (Throwable e) {
+         log.debug("Could not initialize OpenSSL", e);
       }
-      DEFAULT_SSL_PROVIDER = sslProvider;
+      SSL_PROVIDER = sslProvider;
    }
 
    private KeyStore keyStore;
    private String keyStoreFileName;
    private char[] keyStorePassword;
-   private char[] keyStoreCertificatePassword;
    private String keyStoreType = DEFAULT_KEYSTORE_TYPE;
    private String keyAlias;
    private KeyStore trustStore;
    private String trustStoreFileName;
    private char[] trustStorePassword;
    private String trustStoreType = DEFAULT_KEYSTORE_TYPE;
    private String sslProtocol = DEFAULT_SSL_PROTOCOL;
-   private String sslProvider = DEFAULT_SSL_PROVIDER;
    private boolean useNativeIfAvailable = true;
```

Comment:
nitpick, unused.

Reply:
I've completely removed checking for openssl in the static initializer. We now use the ServiceLoader to find providers.